pkg/server/controllers.go

```diff
@@ -362,6 +365,12 @@ func (s *Server) installWorkloadResourceScheduler(ctx context.Context, config *r
 
 func (s *Server) installWorkspaceScheduler(ctx context.Context, config *rest.Config) error {
 	controllerName := "kcp-workspace-scheduler"
+
+	bootstrapConfig := rest.CopyConfig(config)
+	bootstrapConfig = rest.AddUserAgent(kcpclienthelper.SetMultiClusterRoundTripper(bootstrapConfig), controllerName)
+	bootstrapConfig.Impersonate.UserName = "system:kcp:bootstrapper"
+	bootstrapConfig.Impersonate.Groups = []string{bootstrappolicy.SystemKcpWorkspaceBootstrapper}
```

Comment:
no you lost me. Why yet another bootstrap config?

Reply:
adding a comment